### ISSUE TITLE
No reading time for posts

### ISSUE THREAD

**Issue by waja** [Kikobeats/uno-zen#227]
On ghost 1.x the reading time for a post doesn't work anymore:

![reading_time](https://user-images.githubusercontent.com/1449568/28995417-19e59cc2-79e9-11e7-9e2d-2dca576d7817.png)

**Comment by Kikobeats** [comment]
Can you paste devtools state?

**Comment by waja** [comment]
```
uno-zen.common.js?v=a5c0113bdb:8 Uncaught ReferenceError: InstantClick is not defined
    at uno-zen.common.js?v=a5c0113bdb:8
    at HTMLDocument.<anonymous> (zepto.min.js:2)
```
Did completely forget about that.

**Comment by Kikobeats** [comment]
InstantClick is a third party dependency:

https://github.com/Kikobeats/uno-zen/search?utf8=%E2%9C%93&q=InstantClick&type=

I released a new version being sure InstantClick is present, can you confirm me that now is working?

**Comment by waja** [comment]
Jepp. Working well!

Many thanks!

**Comment by waja** [comment]
It also fixed #228.
